modules/deploying-serverless-apps.adoc

```diff
@@ -19,3 +19,8 @@ $ oc apply --filename service.yaml
 Now that service has been created and the application has been deployed, Knative will create a new immutable revision for this version of the application.
 
 Knative will also perform network programming to create a route, ingress, service, and load balancer for your application, and will automatically scale your pods up and down based on traffic, including inactive pods.
+
+[NOTE]
+====
+The first time that a Knative service is created in a namespace, that namespace will automatically receive a new networking configuration. This might cause the initial service to take longer than is usually required for a service to become ready.
+====
```

Comment:
Since we mangle with NetworkPolicy now, let's add:

> If the namespace has no `NetworkPolicy` configuration to it, an "allow all" type policy will be applied automatically. When the last Knative Service is removed from that namespace, this policy will be removed as well if no other `NetworkPolicy` configurations have been applied in the meantime.